packages/app-store/sylapsvideo/DESCRIPTION.md

```diff
@@ -1,6 +1,6 @@
 ---
 items:
-  - /api/app-store/sylaps/sylaps1.png
+  - /api/app-store/sylapsvideo/1.jpg
```

Comment:
This happened I guess because sylaps was renamed in the middle of development and our current logic doesn't remove old files and the wrong path keeps working locally.
<img width="1113" alt="Screenshot 2023-04-12 at 2 42 14 PM" src="https://user-images.githubusercontent.com/1780212/231411308-030dac03-75a5-4aab-8380-9d809fa22909.png">